packages/@o3r/extractors/src/utils/config-doc.ts

```diff
@@ -143,27 +143,26 @@ function getCategoryFromDocText(docComment: string): string | undefined {
 }
 
 /**
- * Get category from a given DocComment.
+ * Get categories from a given DocComment.
  *
- *  The category is extracted from <o3rCategories> tag.
+ * The categories are extracted from the @o3rCategories tags.
  *
- * @param docComment The DocComment to get category from
+ * @param docComment The DocComment to get categories from
  */
-function getCategoriesFromDocText(docComment: string): CategoryDescription[] | undefined {
+export function getCategoriesFromDocText(docComment: string): CategoryDescription[] | undefined {
   const categoriesWithDescription: CategoryDescription[] = [];
-  const categoriesString = docComment.match(/<o3rCategories>(.+)<\/o3rCategories>/s);
+  const categoriesString = Array.from(docComment.matchAll(/@o3rCategories (.*)/g)).map((match) => match[1].trim());
   if (categoriesString) {
-    const categoriesArray = categoriesString[1].match(/<(\w+)>([^<]+)/gs);
-    if (categoriesArray) {
-      for (const catStringWithDesc of categoriesArray) {
-        const catWithDesc = catStringWithDesc.match(/<(\w+)>(.+)/s);
-        if (catWithDesc && catWithDesc[1] && catWithDesc[2]) {
-          const category = catWithDesc[1];
-          const description = catWithDesc[2].replace(/\r?\n \*/g, '').trim();
-          categoriesWithDescription.push({label: description, name: category});
-        } else {
-          throw Error(`Invalid categories description format: ${categoriesString[1]}`);
-        }
+    for (const cat of categoriesString) {
+      const firstSpaceIndex = cat.indexOf(' ');
+      if (firstSpaceIndex < 1) {
```

Comment:
Would be more readable with `firstSpaceIndex === -1` as you trim the `cat` value `firstSpaceIndex` can not be `0`